drivers/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h

```diff
@@ -270,10 +270,10 @@ struct nrf_wifi_fmac_callbk_fns {
 				     signed short signal);
 #endif /* NRF70_STA_MODE */
 #if defined(NRF70_RAW_DATA_RX) || defined(NRF70_PROMISC_DATA_RX)
-	void (*rx_sniffer_frm_callbk_fn)(void *os_vif_ctx,
-					 void *frm,
-					 struct raw_rx_pkt_header *,
-					 bool pkt_free);
+	void (*sniffer_callbk_fn)(void *os_vif_ctx,
```

Comment:
why is `rx_` dropped? This is still applicable only for RX, no?

Reply:
The length of the name is giving issue with the line length adjustments. Since Sniffer is already RX, I chose to keep sniffer.

Thanks and regards
Vivek